MPF.Frontend/Tools/ProtectionTool.cs

```diff
@@ -304,7 +304,82 @@ public static string SanitizeFoundProtections(IEnumerable<string> foundProtectio
+                        .Where(p => !p.StartsWith("Macrovision Protected Application, Macrovision Protected Application [Version Expunged]"))
+                        .Where(p => !(Regex.IsMatch(p, @"SafeDisc [0-9]\.[0-9]{2}\.[0-9]{3}\+", RegexOptions.Compiled)))
+                        .Where(p => !(Regex.IsMatch(p, @"SafeDisc 1\.[0-9]{2}\.[0-9]{3}-[0-9]\.[0-9]{2}\.[0-9]{3}", RegexOptions.Compiled)))
+                        .Where(p => p != "SafeDisc")
+                        .Where(p => p != "SafeDisc 1/Lite")
+                        .Where(p => p != "SafeDisc 2+")
+                        .Where(p => p != "SafeDisc 3+ (DVD)");
+                }
+
+                // Best case for SafeDisc 1.X: A full SafeDisc version is found that isn't part of a version range. 
+                // TODO: Scrub unneeded detections better
+                else if (foundProtections.Any(p => Regex.IsMatch(p, @"SafeDisc 1\.[0-9]{2}\.[0-9]{3}", RegexOptions.Compiled) && !(Regex.IsMatch(p, @"SafeDisc [0-9]\.[0-9]{2}\.[0-9]{3}-[0-9]\.[0-9]{2}\.[0-9]{3}", RegexOptions.Compiled)) && !p.StartsWith("Macrovision Protection File") && !(Regex.IsMatch(p, @"SafeDisc [0-9]\.[0-9]{2}\.[0-9]{3}\+", RegexOptions.Compiled))))
+                {
+                    foundProtections = foundProtections.Where(p => !p.StartsWith("Macrovision Protection File"))
+                        .Where(p => !p.StartsWith("Macrovision Security Driver"))
+                        .Where(p => !(Regex.IsMatch(p, @"SafeDisc [0-9]\.[0-9]{2}\.[0-9]{3}-[0-9]\.[0-9]{2}\.[0-9]{3}", RegexOptions.Compiled)))
+                        .Where(p => !(Regex.IsMatch(p, @"SafeDisc [0-9]\.[0-9]{2}\.[0-9]{3}\+", RegexOptions.Compiled)))
+                        .Where(p => p != "SafeDisc")
+                        .Where(p => p != "SafeDisc 1")
+                        .Where(p => p != "SafeDisc 1/Lite");
+                }
+                
+                // Only SafeDisc Lite is found.
+                else if (foundProtections.Any(p => p == "SafeDisc Lite"))
+                {
+                    foundProtections = foundProtections.Where(p => p != "SafeDisc")
+                        .Where(p => !(Regex.IsMatch(p, @"SafeDisc 1\.[0-9]{2}\.[0-9]{3}-1\.[0-9]{2}\.[0-9]{3}\/Lite", RegexOptions.Compiled)));
+                }
+
+                // Only SafeDisc 2+ is found.
+                else if (foundProtections.Any(p => p == "SafeDisc 2+"))
+                {
+                    foundProtections = foundProtections.Where(p => p != "SafeDisc")
+                        .Where(p => !p.StartsWith("Macrovision Protection File"))
+                        .Where(p => !(Regex.IsMatch(p, @"SafeDisc [0-9]\.[0-9]{2}\.[0-9]{3}\+", RegexOptions.Compiled)));
+                }
+                // TODO: Make sure edge cases, like SafeDisc Lite, are supported properly.
+                // TODO: Organize and re-add the following cases if applicable: 
+                /*
```

Comment:
I know this is still called "WIP" but please don't leave in a large block of commented code.